controller_manager:
  ros__parameters:
    update_rate: 1000  # Hz

    velocity_controller:
      type: velocity_controllers/JointGroupVelocityController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster


velocity_controller:
  ros__parameters:
    joints:
      - omni_back_joint
      - omni_left_joint
      - omni_right_joint